site/docs/releases.md

```diff
@@ -56,8 +56,46 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
+
+High-level features:
+
+* **Core**
+  * Allow Iceberg schemas to specify one or more columns as row identifers [[\#2465](https://github.com/apache/iceberg/pull/2465)]. Note that this is a prerequisite for supporting upserts in Flink.
+  * Added Jdbc [[\#1870](https://github.com/apache/iceberg/pull/1870)] and DynamoDB [[\#2688](https://github.com/apache/iceberg/pull/2688)] Iceberg catalog implementations.
+  * Added predicate pushdown for partitions and files metadata tables [[\#2358](https://github.com/apache/iceberg/pull/2358), [\#2926](https://github.com/apache/iceberg/pull/2926)]
+  * Added a new Compaction API for IcebergV2 and implemented a new compaction strategy [[\#2501](https://github.com/apache/iceberg/pull/2501), [\#2609](https://github.com/apache/iceberg/pull/2609)].
+* **Spark**
+  * [[\#2560](https://github.com/apache/iceberg/pull/2560)] added extensions DDL to set identifier fields 
+  * [[\#2660](https://github.com/apache/iceberg/pull/2660)] added support for micro-batch streaming reads for structured streaming in Spark3
+  * [[\#2777](https://github.com/apache/iceberg/pull/2777)] improves the performance of adding a file to a partition
+* **Flink** now support SQL primary keys [[\#2410](https://github.com/apache/iceberg/pull/2410)]
+
+Important bug fixes:
+
+* **Core**
+  * [\#2849](https://github.com/apache/iceberg/pull/2849) fixes string bucketing with non-BMP characters
+  * [\#2551](https://github.com/apache/iceberg/pull/2551) fixes Parquet dictionary filter not handling fixed length byte arrays.
+  * [\#2550](https://github.com/apache/iceberg/pull/2550) fixes a problem with the configuration of HiveCatalog.
+* **Spark**
+  * [\#2584](https://github.com/apache/iceberg/pull/2584) fixes MERGE INTO in Spark when used with SinglePartition partitioning.
+  * [\#2877](https://github.com/apache/iceberg/pull/2877) fixes nested struct pruning in Spark
+  * [\#2757](https://github.com/apache/iceberg/pull/2757) fixes Timestamp without Timezone type support in Spark
+
+Other notable changes:
+
+* The Iceberg Community [voted to approve](https://mail-archives.apache.org/mod_mbox/iceberg-dev/202107.mbox/%3cCAMwmD1-k1gnShK=wQ0PD88it6cg9mY7Y1hKHjDZ7L-jcDzpyZA@mail.gmail.com%3e) version 2 of the Apache Iceberg Format Specification. The differences between version 1 and 2 of the specification are documented [here](https://iceberg.apache.org/spec/#version-2).
+* Bugfixes and stability improvements for NessieCatalog
+* Improvements and fixes for Iceberg's Python library
+* Added a vectorized reader for Apache Arrow [[\#2286](https://github.com/apache/iceberg/pull/2286)] and modified Iceberg's Hive integration so that Hive can leverage its vectorized ORC reader when accessing Iceberg tables [[\#2613](https://github.com/apache/iceberg/pull/2613)].
+* Spark: Add limited support for vectorized reads for Parquet V2 ([\#2749](https://github.com/apache/iceberg/pull/2749))
```

Comment:
I don't consider this notable. It does not actually support any Parquet v2 encodings, so it isn't useful to anyone.

Reply:
Removed.